From 6b5d9cac9d914ded21435145293cfba03c2e444e Mon Sep 17 00:00:00 2001
From: George Zacharia <george.zcharia@gmail.com>
Date: Tue, 9 Jun 2020 08:09:02 +0530
Subject: [PATCH] DefaultPermissionGrantPolicy: No need to dump the entire
 stacktrace

Currently if an app is not found installed the logcat gets spammed with the entire stacktrace, which is very often in my case, stop that
Change-Id: Ica270065684af4bb199190e46e25267611b4e12e
---
 .../server/pm/permission/DefaultPermissionGrantPolicy.java      | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/core/java/com/android/server/pm/permission/DefaultPermissionGrantPolicy.java b/services/core/java/com/android/server/pm/permission/DefaultPermissionGrantPolicy.java
index 5d97e05ba26..0526d3748e0 100644
--- a/services/core/java/com/android/server/pm/permission/DefaultPermissionGrantPolicy.java
+++ b/services/core/java/com/android/server/pm/permission/DefaultPermissionGrantPolicy.java
@@ -1421,7 +1421,7 @@ public final class DefaultPermissionGrantPolicy {
             return mContext.getPackageManager().getPackageInfo(pkg,
                     DEFAULT_PACKAGE_INFO_QUERY_FLAGS | extraFlags);
         } catch (NameNotFoundException e) {
-            Slog.e(TAG, "PackageNot found: " + pkg, e);
+            Slog.e(TAG, "PackageNot found: " + pkg);
             return null;
         }
     }
